### ISSUE TITLE
Verbatim text

### ISSUE THREAD

**Issue by oysteins** [gap-packages/AutoDoc#159]
If I try to use the GAPDoc element `<Verb>` in an AutoDoc comment, then AutoDoc eats my whitespaces.

Example:
```
#! <Verb>
#! some
#!     verbatim    text
#! </Verb>
```
When I run AutoDoc on the file containing this, it is translated to the following GAPDoc code:
```
<Verb>
some
verbatim text
</Verb>
```
This destroys the whole point of the `<Verb>` element, which is to keep the text exactly as it is, including all whitespaces.

It is even worse if I include a blank line in my verbatim text:
```
#! <Verb>
#! some
#!
#!     verbatim    text
#! </Verb>
```
This causes AutoDoc to insert a "new paragraph" element:
```
<Verb>
some
<P/>
verbatim text
</Verb>
```
I then fails during compilation of the GAPDoc code.

**Comment by sebasguts** [comment]
Yep, that would be possible, and actually the right solution here. Question would be if we do `@Begin/EndVerbatim` or if we include it in the markdown processing, via the GitHub triple-quote
style markdown. @fingolfin, do you have an opinion on this?

**Comment by fingolfin** [comment]
Adding `@Begin/EndVerbatim` seem easiest and would help quickest (and just less than a year after the issue was filed... *ahem*). Triple quote would be nice, too, but one has to carefully decide whether that should be a log, example, listing, verbatim... But perhaps the suggestions on issue #184 would fit in with that (adding a "verbatim" variant there would be no problem).
